admin-dev/themes/new-theme/template/page_header_toolbar.tpl

```diff
@@ -120,5 +121,66 @@
       </ul>
     </div>
   {/if}
+
+  <div class="btn-floating">
+    <button class="btn btn-primary collapsed" data-toggle="collapse" data-target=".btn-floating-container" aria-expanded="false">
+      <i class="material-icons">add</i>
+    </button>
+    <div class="btn-floating-container collapse">
+      <div class="btn-floating-menu">
+        {hook h='displayDashboardToolbarTopMenu'}
+        {foreach from=$toolbar_btn item=btn key=k}
+          {if $k != 'back' && $k != 'modules-list'}
+            {* TODO: REFACTOR ALL THIS THINGS *}
```

Comment:
What do you mean by refactor?

Reply:
I don't know, that's a copy paste from the top of the file, as it's the exact same markup, but on mobile it's inside the floating button

This is not me who wrote this line :/

Reply:
@PierreRambaud

Reply:
Remove these lines if it's not needed and you don't know why they are here :sweat_smile: